pyftpdlib/handlers.py

```diff
@@ -22,6 +22,7 @@
 
 try:
     from OpenSSL import SSL  # requires "pip install pyopenssl"
+    from OpenSSL.SSL import  VERIFY_PEER, VERIFY_FAIL_IF_NO_PEER_CERT, VERIFY_CLIENT_ONCE, SESS_CACHE_OFF, OP_NO_TICKET
```

Comment:
`flake8` (which is run by the Travis builds) is complaining about there being two spaces after `import`